raiden-network/light-client
# Switch wrtc to fork working on M1 processors

Fixes #3135 
Merging to `v2` branch, I'll merge it fixing conflicts to `master` later.

**Short description**
Also, update CLI's README.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. `yarn` on root on an `M1` computer following `brew` instructions on `README`
2. It should install successfuly
3. Will fix the `yarn global add` usecase once released
4. To be forward-ported to `master`
